doc/nrf/releases/release-notes-1.6.0.rst

```diff
@@ -0,0 +1,488 @@
+.. _ncs_release_notes_160:
+
+Changes in |NCS| v1.6.0
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the TF-M, MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices are now supported for production.
+Wireless protocol stacks and libraries may indicate support for development or support for production for different series of devices based on verification and certification status. See the release notes and documentation for those protocols and libraries for further information.
+
+Highlights
+**********
+
+* Added production support for nRF5340 for Bluetooth LE in single protocol configuration and in multiprotocol operation with Thread for Bluetooth Peripheral role.
+* Added production support for multiple advertising sets to SoftDevice Controller.
+* Added features available in Trusted Firmware-M (TF-M) which is supported for development with nRF5340 and nRF9160.
+* Integrated Apple Find My supported for production.
+* Integrated Pelion cloud client supported for development for LTE-M/NB-IoT and Thread.
+* Edge Impulse Machine Learning has been further integrated with an Application example available for Thingy:52.
+* Added complete support for Thread 1.2 mandatory features for routers and end devices supported for development for the nRF52 family.
+* Integrated more complete support for Matter (Project CHIP) which is supported for development for nRF52840 and nRF5340.
```

Comment:
```suggestion
* Matter (formerly Project CHIP) is now supported for development for nRF52840 and nRF5340.
```

Reply:
I think it was already supported for development for nRF52840 and nRF5340.

Reply:
I will leave as is for now, PMT will be looking at this anyway